templates/index.html

```diff
@@ -1,5 +1,5 @@
 {% extends "bootstrap/base.html" %}
-{% block title %}HumanGeo Extra Life{% endblock %}
+{% block title %}{{ team_name }}{% endblock %}
```

Comment:
Should we add a suffix here? i.e. `{{ team_name }} Extra Life`?

Reply:
I can see both sides of this, but my thought was that I'd prefer to leave the slot flexible.

Reply:
Alright. In that case, should we rename the parameter to `page_title`?

Reply:
Yeah I can agree to that. If it was used in additional places as a team name it'd be different, but as it stands `page_title` is more descriptive